packages/README.md

```diff
@@ -0,0 +1,35 @@
+# @blockly Packages
+
+This directory is where [Blockly](http://github.com/google/blockly) extension
+packages such as plugins and fields are authored and stored. These packages are
+released to npm under the ``@blockly`` organization.
```

Comment:
Should we be using @blockly scope instead of organization?

Reply:
Yep!